Patch from
https://git.savannah.gnu.org/cgit/cpio.git/commit/?id=641d3f489cf6238bb916368d4ba0d9325a235afb
has been rebased for 2.13

Sergey Poznyakoff <gray@gnu.org>	2020-01-20 07:45:39 +0200
committer	Sergey Poznyakoff <gray@gnu.org>	2020-01-20 07:45:39 +0200
commit	641d3f489cf6238bb916368d4ba0d9325a235afb (patch)
tree	2911ce855ef9eb6e33a78f9abb6b9b2d5a84cdce
parent	0c4ffde0bdc015b468626f7464e39659c6891a31 (diff)
download	cpio-641d3f489cf6238bb916368d4ba0d9325a235afb.tar.gz
Minor fix
* src/global.c: Remove superfluous declaration of program_name
Diffstat
-rw-r--r--	src/global.c	3	
		
1 files changed, 0 insertions, 3 deletions
diff --git a/src/global.c b/src/global.c
index fb3abe9..acf92bc 100644
--- a/src/global.c
+++ b/src/global.c
@@ -184,9 +184,6 @@ unsigned int warn_option = 0;
 /* Extract to standard output? */
 bool to_stdout_option = false;
 
-/* The name this program was run with.  */
-char *program_name;
-
 /* Extract files over symbolic links */
 bool extract_over_symlinks;

